src/index.js

```diff
@@ -38,19 +38,32 @@ async function getValidPort (port) {
 }
 
 // API
-async function createTestCafe (hostname, port1, port2, sslOptions, developmentMode = false, retryTestPages = false) {
+async function getConfiguration (args) {
```

Comment:
I guess we should leave default values for the `developmentMode` and `retryTestPages` options.

Reply:
Actually, I missed that in the previous review. If they are not specified explicitly, we should take them from the configuration file, don't we?

Reply:
If we specify default values for them here, it will be no longer possible to use values from the configuration file.